include/classes/DAO/DAO.class.php

```diff
@@ -21,14 +21,15 @@
 
 class DAO {
 
-	// private
-	private $db;     // global database connection
-	
+	// protected
+	protected $db;     // global database connection
+	protected $addslashes;
```

Comment:
Is it needed to define `$addslashes` as a variable when DAO classes will be using `$this->addSlashes`?

Also, can the old definition of `$addslashes` in vitals.inc.php be removed?

Reply:
1.  I don't think adding $addslashes should be necessary my return statement in the addSlashes function should have been ```return mysqli_real_escape_string($this->db,$sql)```
2. Yes, I would remove it as it serves no purpose
Thanks for the reviews